src/WebPages/signup/Signup.jsx

```diff
@@ -0,0 +1,139 @@
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
You can just write one rule like this, `import { useState, useEffect } from react`. But, if you like this way you can for it.